cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+\subsubsection{Convection using a pressure--temperature look-up table and the rheology of Steinberger and Calderwood (2006)}
```

Comment:
This is correct as it is: https://www.thepunctuationguide.com/en-dash.html (and I actually learned about when to use an en dash rather than a hyphen while visiting you and reading "How to write a lot" with your working group).

Reply:
:-) Out of curiosity, on that page, you consider "pressure-temperature" as a "connection or conflict"?

Reply:
Yes, it's a connection of two different things (maybe even in the sense of different directions) as opposed to a compound. It's not one thing (a pressure-temperature in the sense that it would be a temperature where the word pressure defines more specifically what kind of temperature it is), but instead a connection of two separate things (pressure and temperature are two different axes in the coordinate system of the table).

Reply:
I learn something new every day :-)